aten/src/ATen/native/native_functions.yaml

```diff
@@ -759,7 +759,7 @@
   device_guard: False
   tags: inplace_view
   dispatch:
-    CompositeExplicitAutograd: as_strided_
+    ZeroTensor, CPU, CUDA, Meta, QuantizedCPU, QuantizedCUDA: as_strided_
```

Comment:
This looks suspicious

Reply:
Oh, I should just make this `CompositeExplicitAutogradNonFunctional`. The main thing is that XLA/LTC shouldn't get a registration for this kernel, since it's problematic for functional backends (which lets us give a better error message if they haven't implemented `as_strided_copy` yet).